src/main/webapp/i18n/en/programmingExercise.json

```diff
@@ -62,6 +62,10 @@
                 "title": "Max Static Code Analysis Penalty",
                 "description": "You can choose a value between 0 and 100 percent or leave the field empty. If the maximum penalty is 20% for a 10 point exercise, a maximum of two points can be deducted because of issues found by static code analysis. If the field is left empty, no penalty limit will be enforced."
             },
+            "recreateBuildPlans": {
+                "title": "Recreate Build Plans",
+                "description": "Activate this option to create new build plans instead of copying them from the imported exercise."
```

Comment:
```suggestion
                "description": "Activate this option to create new build plans instead of copying them from the imported exercise. Newly created build plans support the latest features of the exercise templates by default. We advice you to enable this option, if the exercise to be imported is older than 6 months."
```